test_go_binding.go

```diff
@@ -8,11 +8,7 @@ package mlpack
 */
```

Comment:
It might be worth removing this binding too; it may not be useful for users. :+1:

Reply:
No problem, we can remove this, I just thought that it was useful for testing the `MAKEFILE` file.

Reply:
I think so, after the confirmation on this we can release go-bindings with a release-tag same as mlpack.

Reply:
Sorry for the slow response---it's a holiday weekend in the US so I have been away from a computer. :)  Personally I don't have too much of an opinion, so if you think it's useful for users to have for the sake of testing, we can leave it in.

Reply:
This sounds awesome to have holidays, I guess have you enjoyed the holiday :) :tada: